roles/openstack-setup/tasks/images.yml

```diff
@@ -1,7 +1,12 @@
 ---
+# NOTE: Glance does not know how to proxy copy-from URLs, so we should instead
+# download the image files locally and create images using the file method.
+- name: download images locally
+  get_url: url={{ item.url }} dest=/tmp/{{ item.filename }}
+  with_items: glance.images
 - name: default images
   glance_image: name={{ item.name }}
```

Comment:
Seems like Ansible core module glance_image is deprecated. Should we be switching to os_image assuming we at right Ansible version.